src/GuitarProToMidi.Console/Native/Format.cs

```diff
@@ -284,7 +284,9 @@ private List<Note> RetrieveNotes(GuitarProToMidi.Track track, int[] tuning, Trac
                 case SimileMark.none:
                     break;
                 default:
-                    throw new ArgumentOutOfRangeException(nameof(m.simileMark), m.simileMark, "Unknown enum value.");
+                    Console.WriteLine("Warning: Skipping unhandled data {0} {1} {2}", nameof(m.simileMark), m.simileMark,
```

Comment:
Please use `NLog` instead of simple console prints.

Reply:
You can find an exmple here: https://github.com/rageagainsthepc/GuitarPro-to-Midi/blob/4bfdb1b9dee79c2244971a8a03dc697dfb997b83/src/GuitarProToMidi.Console/GP6File.cs#L402

Reply:
I have updated that :)